README.md

```diff
@@ -23,7 +23,7 @@ Optuna can dynamically construct the search spaces for the hyperparameters.
 
 ## News
 
-- **2021-12-06** First alpha version of Optuna 3.0 is released! Early adopters may want to upgrade and provide feedback for a smoother transition to the coming major release. Try `pip install optuna==3.0.0a0`.
+- **2022-02-14** Pre-releases of Optuna 3.0 are available! Early adopters may want to upgrade and provide feedback for a smoother transition to the coming full release. Find the latest one [here](https://github.com/optuna/optuna/releases). Try `pip install optuna==3.0.0a2`
```

Comment:
> Try `pip install optuna==3.0.0a2`

Isn't it needed anymore? Our release note currently doesn't show the install command so I'm wondering if we can remove it here.

Reply:
Definitely not a strong opinion, but I think we can remove it from here, if we're forwarding to the list of releases. Alternatively, we could show the command with the `--pre` option instead of specifying a concrete value. These ways, we'd not have to update the news after each pre-release (i.e. b0, rc, ..)

Reply:
Thank you for the comments! Both (removing and showing the command with `pre` option) sound reasonable. I personally prefer the latter choice and revise the PR to show the command in 16a788c!